app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<li class="tag">Digital media and information technology</li>
+<li class="tag">Computer networks</li>
+<li class="tag">Cyber security</li>
+<li class="tag">Data representation</li>
+<li class="tag">Digital technology</li>
+</ul>
+<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as a A level qualification and encourage pupils to develop skills to develop websites and prepare them for further learning or employment. Themes you might cover are:</p>
+<ul class="vertical-tags">
+<li class="tag">Abstraction</li>
+<li class="tag">Writing programmes to solve problems</li>
+<li class="tag">Efficiency of algorithms</li>
+<li class="tag">Data structures and the use of databases</li>
+<li class="tag">Systems architecture</li>
+<li class="tag">Computing related mathematics</li>
+</ul>
+
+<p>You can <a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks4-l/units">explore what teaching a computer lesson would be like</a>.</p>
+</section>
+
+  <section class="col col-720">
+    <h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach and it can be in any subject.</p>
+<p>Computer science teacher training courses have had successful applications from candidates with a range of degrees, including business and management, data science, history and economics.</p>
+<p>To train to teach in primary and secondary schools in England, you’ll need:</p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
+<li>a bachelor’s degree in any subject</li>
+</ul>
+<p><a href="/non-uk-teachers">Non-UK citizens can find out more about training to teach in England</a>.</p>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a computing teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge.</p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+
+<h2>Fund your teacher training</h2>
+<p>To train to be a computing teacher, you'll need to do a teacher training course.</p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee computing teachers</a>.</p>
+<p>You can receive this alongside a tuition fee and maintenance loan.</p>
+<p>You may also be able to get <a href="/funding-and-support">extra funding support</a> if you are:</p>
+<ul>
+<li>a parent or carer</li>
+<li>disabled</li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
+<p>You can <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">find out more about funding for non-UK citizens</a>.</p>
```

Comment:
@caitlin-reidy page title is fees and financial support for non-UK trainee teachers so link should ideally match